vectorsearch/workload.json

```diff
@@ -0,0 +1,17 @@
+{% import "benchmark.helpers" as benchmark with context %}
+{
+    "version": 2,
+    "description": "Benchmark for knn_vector field type",
```

Comment:
A more verbose description will be helpful for users not familiar with the domain here.

Reply:
Ack